### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Amrit131**
# Desktop version
### Few functionalities that we added are -:
- Used media query to adjust the website according to screen sizes.
- Added styling to all sections of the website according to  desktop version.
- Positioned element using Grid and Flexbox according to desktop version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,55 +14,124 @@
...
+            </div>    
         </section>
         <section id="projects-container">
+            <div class="desktop-projects">
+                <div class="see-projects">
+                    <img src="images/projects-bg-1.png" alt="See Project background image" id="project-bg">
+                    <div id="btn-see-project">
+                        <button id="see-project-btn">See Projects</button>
+                    </div>
+                </div>
```
Comment (codecaiine) [inline index.html]: - [x] Good work so far your project is very impressive :clap: But I noticed that part of the project with the description and dark opacity is static. Please make sure that disappears when the project is hovered on with the mouse it disappears and we have a `See Project` button appears at the bottom and leads to the Live Demo of the project or to the GitHub repository. :+1: . Please fix it in order to give a good UI/UX experience and make your website wonderful :smile:

File: style.css [hunk 2]
```diff
@@ -611,3 +620,807 @@ body {
   list-style: none;
   padding-left: 80px;
 }
+
+/* Desktop version */
+
+@media screen and (min-width: 768px) {
```
Comment (codecaiine) [inline style.css:626]: - [x] I noticed that your design has some issues that you will need to fix. Please make your website and mobile form adapt well to any kind and size of desktop and mobile screen. You can see the issues with the video recording that I'm sharing with you below :arrow_down: It's really important to pay attention to the details :smile: 

https://www.loom.com/share/0b8f8edf20794ad68fb931f1ca3a24c0

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Amrit131 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Great an impressive UI design
- No linters errors :heavy_check_mark: 
- Professional Repo :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Amrit131) [reply index.html]: I have implemented the given suggestion thank you.

File: style.css [hunk 3]
```diff
@@ -611,3 +620,808 @@ body {
...
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  ::-webkit-scrollbar-bottom {
+    display: none;
+  }
```
Comment (Abenezer-Tilahun) [inline style.css]: - [x] As stated in the project requirement and suggested by the previous reviewer You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Project design has still the same issues : 
Capture

- [x] You can also refer the last reviewer Loom video(https://www.loom.com/share/0b8f8edf20794ad68fb931f1ca3a24c0)

File: style.css [hunk 4]
```diff
@@ -611,3 +620,808 @@ body {
...
+
+  .form-container {
+    width: 700px;
+    height: 630px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 200px;
+    margin-left: 40px;
+  }
```
Comment (Abenezer-Tilahun) [inline style.css]: - [ Optional ] The Minimum mobile screen size should be 375px , so it makes responsive your design I recommend you to adjust the alignment of the Forms for contact section just like the Figma design . 

Your design for contact section : Form container look likes this : 
Capture

The Figma design Look likes this : 
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Amrit131  , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional   README
✔️ Good commit messages

almost

## Required Changes ♻️
_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (Amrit131) [reply style.css:626]: I have implemented the given feedback.

Comment (christianbiring1) [reply style.css:626]: - [ ] Kindly make sure you fix the responsiveness issue as suggested by the previous reviewers.
As you can you how your design is looking :point_down: 
Screenshot from 2022-09-30 16-50-37
- [ ] Kindly make sure you fix this issue
[here](https://www.loom.com/share/e81ca027adb84fcdab6113f47d54e730) you can see how your page is responding on differents screen size

File: style.css [hunk 5]
```diff
@@ -611,3 +622,837 @@ body {
...
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    background-color: #3c3c3c;
+  }
+
+  #heading {
+    width: 20%;
+    display: flex;
+    align-items: center;
```
Comment (christianbiring1) [inline style.css:645]: - [ ] Kindly make sure your page is not overlapping on the right on the movie version. The content must fit all the available width of the actual page.
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Amrit131 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# STATUS: REQUEST CHANGES :recycle: 
almost


 ## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Good commit history
- [ ] Descriptive PR `Title`

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 5 above)
Comment (Amrit131) [reply style.css:645]: @christianbiring1 I do not understand this issue can you please elaborate more ?

**Review by Alfredbis29 — APPROVED** [review]
Hi @Amrit131   , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**